View/Helper/WizardHelper.php

```diff
@@ -118,12 +125,12 @@ public function progressMenu($titles = array(), $attributes = array(), $htmlAttr
 				if ($step == $activeStep) {
 					$class .= ' active';
 				}
-				$this->output .= "<$wrap class='$class'>" . $this->Html->link($title, array(
+				$this->output .= "<$wrap class=\"$class\">" . $this->Html->link($title, array(
 						'action' => $wizardAction,
 						$step
 					), $htmlAttributes, $confirmMessage) . "</$wrap>";
 			} else {
-				$this->output .= "<$wrap class='incomplete'>" . $title . "</$wrap>";
+				$this->output .= "<$wrap class=\"incomplete\">" . $title . "</$wrap>";
```

Comment:
`"<$wrap class=\"incomplete\">"` results in `<div class="incomplete">`.
`'<$wrap class=\"incomplete\">"` will result in `<$wrap class="incomplete">`.